extension-workspace-references.md

```diff
@@ -4,6 +4,65 @@ The `workspace/xreferences` extension to the Language Server Protocol (LSP) enab
 
 Use case: clients of a language server can invoke `workspace/xreferences` in order to find references to dependencies. This information can then be stored in a database, which allows the caller to create a "global mapping" of symbols in dependencies to the workspaces they are used in (e.g. to see "how do other people use this symbol?"). A user would perform `textDocument/xdefinition` in order to locate the metadata about the symbol they are interested in and find its references in the database.
 
+### New Identifiers
+
+#### Packages
+
+The extension adds the concept of "packages" to LSP.
+A package is specifically something that would usually be installed through a package manager like npm, Composer or Maven.
+A workspace can contain multiple packages in the form of own packages and dependencies.
+A package is identified by a language-specific interface, that is defined by the language server:
```

Comment:
Don't use comma with "that" construction: "A package is identified by a language-specific interface that is defined by the language server:"